front-end/src/app/shared/pipes/report-code-label-list.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ReportCodeLabelList, ReportCodeLabel } from '../utils/reportCodeLabels.utils';
+
+@Pipe({ name: 'findOnReportCodePipe' })
+export class FindOnReportCodePipe implements PipeTransform {
+  transform(list: ReportCodeLabelList | null, reportCode: string) {
+    let label: ReportCodeLabel = { label: '', report_code: '' };
+    let found: ReportCodeLabel | undefined = undefined;
+    if (list != null) {
```

Comment:
could you combine these two if statements into `if (list?.length > 0)`.  The code is generally easier to understand if there are less conditional scopes